src/app/(layout-with-navigation)/(simulation)/simulateur/[...dottedName]/_components/form/Navigation.tsx

```diff
@@ -24,20 +24,22 @@ export default function Navigation({ question }: Props) {
   const router = useRouter()
   return (
     <div className="flex justify-end  gap-4">
-      <Button
-        disabled={noPrevQuestion && noPrevCategory}
-        onClick={() => {
-          if (noPrevQuestion) {
-            if (!noPrevCategory) {
-              gotoPrevCategory()
+      {!(noPrevQuestion && noPrevCategory) ? (
+        <Button
+          disabled={noPrevQuestion && noPrevCategory}
+          onClick={() => {
+            if (noPrevQuestion) {
```

Comment:
Pas fan des if imbriqués 😕 que penses-tu de :

```suggestion
            if (!noPrevQuestion) {
            gotoPrevQuestion()
            return
            }
            if (!noPrevCategory) {
                            gotoPrevCategory()
              }
            
```